numerapi/numerapi.py

```diff
@@ -112,6 +114,73 @@ def download_current_dataset(self, dest_path=".", dest_filename=None,
 
         return dataset_path
 
+    def get_latest_data_path(self, data_type: str, ext: str = "csv") -> str:
+        """Fetch url of the latest data path for a specified data type
+
+        Args:
+            data_type (str): type of data to return
+            data_type (str): file extension to get (optional, defaults to csv)
+
+        Returns:
+            str: url of the requested dataset
+
+        Example:
+            >>> NumerAPI().get_latest_data_path("live", "csv")
```

Comment:
in `utils` there is function to download files from urls. Maybe add that to the example like `numerapi.utils.dowload_file(url)`